fvm/src/call_manager/default.rs

```diff
@@ -294,7 +295,19 @@ where
             };
 
             // Make a store.
+            let gas_available = kernel.gas_available();
```

Comment:
Prefer to get the gas available from the gas tracker directly (owned by the CallManager) outside the closure. Getting it from the kernel at this stage is confusing.

Reply:
I don't think that works because we store the params inside the kernel which can lead to a change in gas_available -- am I missing something?